djweb/dj_comp_hist/views.py

```diff
@@ -197,17 +206,18 @@ def generate_search_facets(doc_objs):
         for organization in doc.recipient_organization.all():
             counter_recipient_organizations[organization.name]=+1
         for cc in doc.cced_person.all():
-            counter_cc[cc.fullname]=+1
-        ##cced organization
-    print(counter_authors)
-    dict_facets={"authors":counter_authors.most_common(10), #10 most common ones
-                 "recipients":counter_recipients.most_common(10),
-                 "years":counter_dates.most_common(10),
-                 "author organizations":counter_author_organizations.most_common(10),
-                 "recipient organizations":counter_recipient_organizations.most_common(10),
-                 "cced":counter_cc.most_common(10)}
-    print(dict_facets)
-
+            counter_cceds[cc.fullname]=+1
+        for organization in doc.cced_organization.all():
+            counter_cced_organizations[organization.name]=+1
+    dict_facets={"authors":dict(counter_authors.most_common(10)),
+                 "author organizations": dict(counter_author_organizations.most_common(10)),
+                 "recipients":dict(counter_recipients.most_common(10)),
+                 "recipient organizations": dict(counter_recipient_organizations.most_common(10)),
+                 "cceds":dict(counter_cceds.most_common(10)),
+                 "cced organizations":dict(counter_cced_organizations.most_common(10)),
+                 "years": dict(counter_dates.most_common(10)),
+                 }
+    return dict_facets
```

Comment:
a little more spacing would be nice here. if you could look for the yellow squiggly lines and follow their conventions that'd be nice! otherwise no pressure